catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -246,10 +247,10 @@ public SourceResponse query(QueryRequest request) throws UnsupportedQueryExcepti
 
         SourceResponse registryQueryResponse = super.query(request);
         for (Result singleResult : registryQueryResponse.getResults()) {
-            if (singleResult.getMetacard()
-                    .getAttribute(RegistryObjectMetacardType.REGISTRY_ID)
-                    .getValue()
-                    .toString()
+            if (!RegistryUtility.isRegistryMetacard(singleResult.getMetacard())) {
```

Comment:
We weren't checking REGISTRY_TAG before, do we need the isRegistryMetacard check?